queries/rescript/highlights.scm

```diff
@@ -66,21 +77,42 @@
 [
  (parameter (value_identifier))
  (labeled_parameter (value_identifier))
-] @parameter
+] @variable.parameter
 
 ; single parameter with no parens
-(function parameter: (value_identifier) @parameter)
+(function parameter: (value_identifier) @variable.parameter)
```

Comment:
This and other similar changes conform to to Neovim's new tags with fallbacks. Meaning, if a certain theme specifies a color for `@variable.parameter`, it will be used, but otherwise it will fall back to the color for `@variable`.